SETUP.md

```diff
@@ -156,9 +158,31 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
     1. `bundle exec rake install` must always be called from the local project's root directory, or it won't work.
     1. Finally, don't worry if your versions don't match the versions in the overview if you're following this method; the installation should still work properly regardless
 
-### Windows note: use an Ubuntu VM
+### Windows
+
+Windows Subsystem for Linux (WSL) allows you to run a GNU/Linux environment directly on Windows without the overhead of a virtual machine. This is the easiest way to get Ruby and other prerequisites running on Windows.
+
+It is worthwhile to make sure that you are using WSL 2. Attempting to use WSL 1 in the past resulted in errors with mysql and pdftk installation. In order to use WSL 2, you must be running Windows 10, updated to version 2004, Build 19041 or higher. If your Windows update service doesn't give you the update automatically, you can download it [here](https://www.microsoft.com/en-us/software-download/windows10).
+
+1. Enable WSL (unabridged instructions [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10)). You should run Powershell as Administrator for the following commands:
+    1. `dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart`
+    1. `dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart`
+    1. Restart your machine. WSL 2 will be the default if your Windows version is sufficiently updated.
+    1. `wsl --set-default-version 2`
+        1. You may need to [update the WSL 2 Linux kernel](https://docs.microsoft.com/en-us/windows/wsl/wsl2-kernel)
+1. [Install Ubuntu 20.04](https://www.microsoft.com/store/productId/9NBLGGH4MSV6) (Windows Store link)
+    * If you want to follow the Ubuntu setup exactly, Ubuntu 16.04 is available from the [Microsoft docs](https://docs.microsoft.com/en-us/windows/wsl/install-manual).
+1. Make sure virtualization is turned on your BIOS settings.
+1. Run `wsl` from the command line (or search for 'Ubuntu' in the Start Menu) for the first time and WSL will complete installation in the resulting terminal window.
```

Comment:
Suggest re-wording
"From the command line, run `wsl`, or from the start menu, find and start 'Ununtu'. When this runs, WSL will complete the installation in the resulting terminal window.